lib/watcher/cpu-profile.js

```diff
@@ -5,7 +5,7 @@
 
 'use strict';
 
-var agent = require('../agent');
+var agent = require('../agent')();
```

Comment:
Do you mean this needs to add some lazy initialization like in adapter.js?  Or just move the brackets out of the require statement?